terraform/modules/iam/variables.tf

```diff
@@ -120,3 +120,8 @@ variable "reference_counter_check" {
   description = "The IAM role for checking DynamoDb reference counter table"
   default     = true
 }
+
+variable "inspector" {
+  description = "The role for retrieving AWS inspector findings"
```

Comment:
Shouldn't this be something like `"Whether or not to create a role for retrieving AWS inspector findings"`, unless I'm misinterpreting what this does

Reply:
That is what it's for. That's a better description. I've updated it.